flink-1.17-connector-bigquery/flink-connector-bigquery/src/test/java/com/google/cloud/flink/bigquery/table/BigQueryDynamicTableSinkITCase.java

```diff
@@ -45,10 +49,11 @@
 import java.io.IOException;
 import java.time.Instant;
 import java.util.List;
-import java.util.Map;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+import static com.google.cloud.flink.bigquery.fakes.StorageClientFaker.createConnectOptionsForWrite;
+
 /** An integration test for the SQL interface of the BigQuery connector. */
 public class BigQueryDynamicTableSinkITCase {
```

Comment:
Please use verbose names, this is a integration test so have that in the name. IT is very cryptic.

Reply:
Fixed.